test/cpp/lite_interpreter_runtime/resources.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <experimental/filesystem>
+#include <string>
+
+namespace torch {
+namespace testing {
+
+/// Gets the path to the resource identified by name.
+///
+/// @param name identifies a resource, relative path starting from the
+///             repo root
+inline auto getResourcePath(std::string name)
+    -> std::experimental::filesystem::path {
```

Comment:
Isn't it C++17 feature (and we still stuck with C++14 compatibility?)

Reply:
Without the experimental it would be a C++17 feature, yes.